samples/azure-quantum/grover/README.md

```diff
@@ -0,0 +1,47 @@
+---
+page_type: sample
+author: cgranade
+description: Search unstructured data on quantum hardware, using the Azure Quantum service
+ms.author: chgranad@microsoft.com
+ms.date: 01/25/2021
+languages:
+- qsharp
+products:
+- qdk
+- azure-quantum
+---
+
+# Simple Grover's search
```

Comment:
Recommend you add brand - it will help search rank.
"Simple Grover's search on Azure Quantum" - or something like that.